### DISCUSSION TITLE
Installation guide for Linux distros

### DISCUSSION THREAD

**Discussion by aerickt** [opensteno/plover#1282] [General]
Since the AppImage is the preferred way to install Plover on Linux, should we remove the PPA for Ubuntu? I'm also thinking of revising the guide so that the AppImage method will be shown first.

**Comment by user202729** [comment]
Good point. Support for Linux packages are supposedly dropped as well (PR 1240 )

**Reply by aerickt** [reply]
I'm not sure if it's a good idea to wait until 4.0.0 is released or just edit it right now though. But then again, I don't think there's any reason to be using Plover v3.

**Comment by parnikkapore** [comment]
I didn't even know that there was a PPA, I just installed Plover straight from pip!

**Comment by aerickt** [comment]
Also, if I'm not mistaken, I think the PPA is unmaintained at this point?

**Reply by benoit-pierre** [reply]
Yes, we might as well remove any mention of it right now.

**Comment by aerickt** [comment]
I've now edited the page to remove the PPA and "other methods of installing Plover".
